pkgs/by-name/fr/frog-protocols/package.nix

```
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  meson,
+  ninja,
+  nix-update-script,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "frog-protocols";
+  version = "0.01-unstable-2024-09-25";
+
+  src = fetchFromGitHub {
+    owner = "misyltoad";
+    repo = "frog-protocols";
+    rev = "17be81da707722b4f907c5287def442351b219b0";
+    hash = "sha256-N8a+o5I7CRoONCvjMHVmPkJTVncczuFVRHEtMFzMzss=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+
+  passthru.updateScripts = nix-update-script { };
+
+  meta = {
+    description = "Wayland protocols but much more iterative";
+    homepage = "https://github.com/misyltoad/frog-protocols";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ Scrumplex ];
+    mainProgram = "frog-protocols";
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = lib.platforms.all;
    pkgConfigModules = [ "frog-protocols" ];
```
This is used for the above test